client/components/post-card/style.scss

```diff
@@ -218,10 +218,6 @@
 	padding-left: 20px;
 }
 
-.post-card__search .like-button .gridicon {
-	top: 2px;
-}
-
 .post-card__search .comment-button .gridicon {
```

Comment:
@mtias just removed it in: https://github.com/Automattic/wp-calypso/pull/7079/commits/9944436268dd0fc2e1d327023c32a973aefbf059, checked and it didn't make any difference.